README.md

```diff
@@ -225,20 +225,38 @@ Example webhook definition
 }
 ```
 
+## Download webhook definitions and webhook payloads schema
+
+You can download the latest `index.json` and `schema.json` files from [unpkg](https://unpkg.com/browse/@octokit/webhooks-definitions@3.24.0/)
+
+- [`index.json`](https://unpkg.com/@octokit/webhooks-definitions/index.json)
+- [`schema.json`](https://unpkg.com/@octokit/webhooks-definitions/schema.json)
+
 ## Usage as Node module
 
+To get an array of all webhook definition objects, require the `index.json` file
+
+```js
+// Use Node.js require:
+const WEBHOOKS = require("@octokit/webhooks-definitions/index.json");
+
+// Or ESM/TypeScript import:
+import WEBHOOKS from "@octokit/webhooks-definitions/index.json";
 ```
-const WEBHOOKS = require('@octokit/webhooks-definitions')
-```
 
-returns an array of webhook definition objects, see example above.
+To get the JSON schema for webhook payloads, require the `schema.json` file
+
+```js
+// Use Node.js require:
+const SCHEMA = require("@octokit/webhooks-definitions/schema.json");
+
+// Or ESM/TypeScript import:
+import SCHEMA from "@octokit/webhooks-definitions/schema.json";
+```
 
 ## How it works
 
-This package updates itself using a daily cronjob running on GitHub Actions. The
-[`index.json`](index.json) file is
-generated by [`bin/octokit-webhooks.js`](bin/octokit-webhooks.js). Run
-`node bin/octokit-webhooks --usage` for instructions.
+This package updates itself using a daily cronjob running on GitHub Actions. The [`index.json`](index.json) file is generated by [`bin/octokit-webhooks.js`](bin/octokit-webhooks.js). Run `node bin/octokit-webhooks --usage` for instructions. After the update is complete, run `npm run build` to update the `index.json` and `schema.json` files.
```

Comment:
Please see my comment on `package.json`

This currently doesn't update `index.json`

Reply:
```suggestion
This package updates itself using a daily cronjob running on GitHub Actions. The [`index.json`](index.json) file is generated by [`bin/octokit-webhooks.js`](bin/octokit-webhooks.js). Run `node bin/octokit-webhooks --usage` for instructions. After the update is complete, run `npm run build:schema` to update `schema.json`.
```